yarn.lock

```diff
@@ -415,6 +437,11 @@ locate-path@^3.0.0:
     p-locate "^3.0.0"
     path-exists "^3.0.0"
 
+lodash@^4.16.4:
```

Comment:
Looks like you need to run `npx yarn-deduplicate` here? As there are 2 lodash versions now.

Reply:
Fixed.